bindings/python/visualize/meshcat_visualizer.py

```diff
@@ -392,6 +402,40 @@ def set_primitive_geometry_transform(
             transform[3, 3] = 1
             self.viewer[shape_name].set_transform(transform)
 
+    def set_arrow_transform(self, origin, vector, shape_name="iDynTree"):
+        if not self.__arrow_exists(shape_name):
+            warnings.warn("The arrow named: " + shape_name + " does not exist.", category=UserWarning, stacklevel=2)
+            return
+
+        # compute the scaling matrix
+        S = np.diag([1, 1, np.linalg.norm(vector)])
+        transform = np.zeros((4, 4))
+        transform[0:3, 3] = np.array(origin) + np.array(vector) / 2
+        transform[3, 3] = 1
+
+        if np.linalg.norm(vector) < 1e-6:
+            warnings.warn("The vector is too small to be visualized.", category=UserWarning, stacklevel=2)
```

Comment:
Is there something less than warning? It often happens that you don't care too much about the length of the arrow before displaying it. Having a warning continuously printed might be annoying

Reply:
I agree, if you display data from a sensor (i.e. force sensor) it makes total sense to have a near 0 value, there is any other reason to have this print/check?